app/views/home/index.html.erb

```diff
@@ -5,8 +5,10 @@
   <p>It comes with many very good features.</p>
   <p class="lead">
     <%# fa_icon helper: https://github.com/bokmann/font-awesome-rails %>
-    <%= link_to (fa_icon 'sign-in', text: 'Log-In'), new_user_session_path, class: 'btn btn-primary btn-lg' %>
-    <%= link_to (fa_icon 'user-plus', text: 'Sign-Up'), new_user_registration_path, class: 'btn btn-secondary btn-lg' %>
+    <% if user_signed_in? == false %>
+      <%= link_to (fa_icon 'sign-in', text: 'Log-In'), new_user_session_path, class: 'btn btn-primary btn-lg' %>
+      <%= link_to (fa_icon 'user-plus', text: 'Sign-Up'), new_user_registration_path, class: 'btn btn-secondary btn-lg' %>
+    <% end %>
```

Comment:
could you build a nice if else here instead

Reply:
did it